Makefile

```diff
@@ -471,6 +471,7 @@ test-unit-python-standalone: ## Run Python unit tests (standalone)
 		/bin/bash -c "cmake -DBUILD_PYTHON_BINDINGS=ON -DBUILD_UNIT_TESTS=OFF .. \
 		&& $(MAKE) -j 4 && python3.11 -m pip install --root-user-action=ignore bindings/python/dist/*311*.whl \
 		&& python3.11 -m pip install plotly pandas \
+		&& python3.11 -m pip install git+https://github.com/lucas-bremond/cesiumpy.git#egg=cesiumpy \
```

Comment:
Can do in a separate MR but should we just make a `test-requirements.in`?

Reply:
Yeah that'd be great!